README.md

```diff
@@ -66,6 +66,19 @@ An example of this can be found in the [Decentraland repository](https://github.
     yarn build      # Will drop the results in dist/
     yarn build-ipfs # Will also deploy to IPFS and output an IPFS hash
     ```
+9.  Deploy your subgraph to one of the hosted nodes. The following command builds and deploys the subgraph continuously as you are making changes to it:
+    ```sh
+    graph \
+       --watch \
+       --verbosity debug \
+       --node $GRAPH_NODE_MANAGEMENT \
```

Comment:
These environment variables are never mentioned anwyhere else. Let's add a `"deploy"` script in step 5, with specific values for the arguments and then mention that here.

Reply:
The env vars are still there though. Let's instead use real values in this example.